tools/cli/help.txt

```diff
@@ -914,11 +914,11 @@ Options:
 Publish a new version of a package to the package server.
 
 Usage: meteor publish [--create]
-       meteor publish --update
+       meteor publish --update [--release <version>]
```

Comment:
I'm wondering if `--release` should be moved to the first example. It's normally used when publishing a new version instead of when updating the metadata.

Reply:
I agree, especially now, using `--release` might become more necessary

Reply:
Why not both? It applies to both anyway.

Reply:
correct, I used it a lot when I published packages for dual Meteor 2+3 compatibility but I had to use `--release 3.0.0-rc.2` explcitly to make it work because at that time the top-level Meteor version was still 2.x